lib/logstash/inputs/eventlog.rb

```diff
@@ -89,6 +93,8 @@ def run(queue)
 
       end # while
 
+    rescue LogStash::ShutdownSignal
+
```

Comment:
I don't think we need to catch logStash::ShutdownSignal here. What we really need is to make the rescue block only catch eventlog error and let the exception bubble up to the pipeline.